app/ios/Podfile

```diff
@@ -26,5 +26,16 @@ target 'STA' do
 
   post_install do |installer|
     react_native_post_install(installer)
+    # Add these lines for Xcode 14 builds
+    installer.generated_projects.each do |project|
+      project.targets.each do |target|
+          target.build_configurations.each do |config|
+              config.build_settings["DEVELOPMENT_TEAM"] = "TEAM ID" 
```

Comment:
Sorry have just seen this! Was a bit concerned that if it was hardcoded it could invalidate the ID (same way as the Jira Api password is wiped) - could be worth a try though!

Reply:
Ah didn't think of that, good point Cat. Better to add it to the .env in the same was as the individual APPLE_ID and include it to the iOS deployment instructions.

Reply:
Hey @PammyLo have being doing a bit of messing about trying to get it to pull from the .env file that sits in the App folder but it always breaks the pod install command...not too sure if it's possible to do it that way (or if I'm just lacking the knowledge!)